src/TTabBar.cpp

```diff
@@ -272,7 +272,7 @@ void TTabBar::dropEvent(QDropEvent* event)
     const QMimeData* mimeData = event->mimeData();
     if (mimeData->hasFormat("application/x-mudlet-tab")) {
         const QString tabName = QString::fromUtf8(mimeData->data("application/x-mudlet-tab"));
-        const int dropIndex = tabAt(event->pos());
+        const int dropIndex = tabAt(event->position().toPoint());
```

Comment:
Since `QDropEvent::post()` was deprecated in Qt 6.0 and we require at least that there is no need for any sort of version check for this one.

Reply:
Is this an actionable comment or simply an observation? Thanks.

Reply:
Just an observation - relief that we don't have to worry about the Qt version.